moped-database/migrations/1700067187681_add_zicla_barriers_subcomponent/down.sql

```diff
@@ -0,0 +1,12 @@
+DELETE FROM moped_components_subcomponents WHERE id IN (
+    SELECT
+        mcs.id
+    FROM 
+        moped_components_subcomponents mcs 
+    JOIN 
+        moped_subcomponents ms ON ms.subcomponent_id = mcs.subcomponent_id
+    WHERE 
+        subcomponent_name = 'Protection Type - ZICLA Barriers');
```

Comment:
it's pretty annoying that you aren't allowed to delete directly from `moped_subcompoents` and let it cascade.

```sql
delete from moped_subcomponents where subcomponent_name = 'Protection Type - ZICLA Barriers';
```

if you wanted to change that `on delete` rule i would support it.